package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "2.1.2",
+  "version": "2.1.3",
```

Comment:
I'm not sure how to go about this, but IMO Coop IAs should increase the major number, and so should Dissolutions.

Reply:
Very good point, we should developing dissolutions at 2.2.x at the very least eh?

Reply:
Since we're done with Coop IAs (I think), let's just go to 3.0.0 ? 🤷‍♂️

Reply:
Sure, lets go!

Reply:
What this means is that the final version of a completed feature might be, say, 2.1.3 (or later if we have to bugfix it after deployment), while the initial (draft/alpha) version of a new feature might be, say, 3.0.0. That seems OK to me :)